setup.py

```diff
@@ -30,7 +31,7 @@ def read(fname):
 
 setup(
     name="mbed-os-tools",
-    version="0.0.1",
+    version=mbed_os_tools.__version__,
```

Comment:
This may not work, as we may not have an `mbed_os_tools` module when running this.

Reply:
I tried this locally by doing a `python setup.py install` and it seemed to work fine.

No mbed-os-tools present:
```
$ pip list
<snip>
mbed-greentea                 1.4.0       c:\users\bridan01\onedrive_arm\documents\dev\greentea
mbed-host-tests               1.4.2       c:\users\bridan01\onedrive_arm\documents\dev\htrun
mbed-ls                       1.6.2
mccabe                        0.6.1
<snip>
```

Installing from github (maybe this simulates pypi installs better than local installs?):
```
$ pip install git+https://github.com/bridadan/mbed-os-tools.git@version_into_module
Collecting git+https://github.com/bridadan/mbed-os-tools.git@version_into_module
  Cloning https://github.com/bridadan/mbed-os-tools.git (to revision version_into_module) to c:\users\bridan01\appdata\local\temp\pip-req-build-nhovio
Requirement already satisfied: PySerial>=3.0 in c:\python27\lib\site-packages (from mbed-os-tools==0.0.1) (3.4)
Requirement already satisfied: requests in c:\python27\lib\site-packages (from mbed-os-tools==0.0.1) (2.20.0)
Requirement already satisfied: pyOCD==0.12.0 in c:\python27\lib\site-packages (from mbed-os-tools==0.0.1) (0.12.0)
Requirement already satisfied: intelhex in c:\python27\lib\site-packages (from mbed-os-tools==0.0.1) (2.2.1)
Requirement already satisfied: future in c:\python27\lib\site-packages (from mbed-os-tools==0.0.1) (0.16.0)
Requirement already satisfied: PrettyTable>=0.7.2 in c:\python27\lib\site-packages (from mbed-os-tools==0.0.1) (0.7.2)
Requirement already satisfied: fasteners in c:\python27\lib\site-packages (from mbed-os-tools==0.0.1) (0.14.1)
Requirement already satisfied: appdirs>=1.4 in c:\python27\lib\site-packages (from mbed-os-tools==0.0.1) (1.4.3)
Requirement already satisfied: junit-xml in c:\python27\lib\site-packages (from mbed-os-tools==0.0.1) (1.8)
Requirement already satisfied: lockfile in c:\python27\lib\site-packages (from mbed-os-tools==0.0.1) (0.12.2)
Requirement already satisfied: six in c:\python27\lib\site-packages (from mbed-os-tools==0.0.1) (1.11.0)
Requirement already satisfied: colorama<0.4,>=0.3 in c:\python27\lib\site-packages (from mbed-os-tools==0.0.1) (0.3.9)
Requirement already satisfied: chardet<3.1.0,>=3.0.2 in c:\python27\lib\site-packages (from requests->mbed-os-tools==0.0.1) (3.0.4)
Requirement already satisfied: certifi>=2017.4.17 in c:\python27\lib\site-packages (from requests->mbed-os-tools==0.0.1) (2018.4.16)
Requirement already satisfied: urllib3<1.25,>=1.21.1 in c:\python27\lib\site-packages (from requests->mbed-os-tools==0.0.1) (1.23)
Requirement already satisfied: idna<2.8,>=2.5 in c:\python27\lib\site-packages (from requests->mbed-os-tools==0.0.1) (2.6)
Requirement already satisfied: enum34 in c:\python27\lib\site-packages (from pyOCD==0.12.0->mbed-os-tools==0.0.1) (1.1.6)
Requirement already satisfied: websocket-client in c:\python27\lib\site-packages (from pyOCD==0.12.0->mbed-os-tools==0.0.1) (0.47.0)
Requirement already satisfied: intervaltree in c:\python27\lib\site-packages (from pyOCD==0.12.0->mbed-os-tools==0.0.1) (2.1.0)
Requirement already satisfied: pyelftools in c:\python27\lib\site-packages (from pyOCD==0.12.0->mbed-os-tools==0.0.1) (0.24)
Requirement already satisfied: pywinusb>=0.4.0 in c:\python27\lib\site-packages (from pyOCD==0.12.0->mbed-os-tools==0.0.1) (0.4.2)
Requirement already satisfied: monotonic>=0.1 in c:\python27\lib\site-packages (from fasteners->mbed-os-tools==0.0.1) (1.5)
Requirement already satisfied: sortedcontainers in c:\python27\lib\site-packages (from intervaltree->pyOCD==0.12.0->mbed-os-tools==0.0.1) (2.0.5)
pyopenssl 18.0.0 has requirement cryptography>=2.2.1, but you'll have cryptography 2.1.4 which is incompatible.
Installing collected packages: mbed-os-tools
  Running setup.py install for mbed-os-tools ... done
Successfully installed mbed-os-tools-0.0.1
You are using pip version 10.0.1, however version 18.1 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.

$ pip list
<snip>
mbed-greentea                 1.4.0       c:\users\bridan01\onedrive_arm\documents\dev\greentea
mbed-host-tests               1.4.2       c:\users\bridan01\onedrive_arm\documents\dev\htrun
mbed-ls                       1.6.2
mbed-os-tools                 0.0.1
mccabe                        0.6.1
<snip>

$ python
Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:42:59) [MSC v.1500 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import mbed_os_tools
>>> mbed_os_tools.__version__
'0.0.1'
```

Reply:
It seems like when the package is being installed, the source of the package is added to the PYTHONPATH, but I'm not 100% sure on that.

Reply:
Thanks for checking.

Reply:
If it doesn't work, I'm sure we'll find out soon enough 😛